modules/installation-configuration-parameters.adoc

```diff
@@ -677,7 +677,11 @@ ifdef::gcp[If you are installing on GCP into a shared virtual private cloud (VPC
 ====
 Not all CCO modes are supported for all cloud providers. For more information about CCO modes, see the _Cloud Credential Operator_ entry in the _Cluster Operators reference_ content.
 ====
-|`Mint`, `Passthrough`, `Manual`, or an empty string (`""`).
+[NOTE]
+====
+If your AWS account has SCP (Service control policies) enabled, you must configure the `credentialsMode` parameter to `Mint`, `Passthrough` or `Manual`.
```

Comment:
I think you should s/SCP (Service control policies)/service control policies (SCP)

Note the lower-case `s` in `service`.
See: https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_inheritance_auth.html

Reply:
Addressed